base/src/main/java/proguard/evaluation/value/InstructionOffsetValue.java

```diff
@@ -87,15 +96,7 @@ public int instructionOffset(int index)
      */
     public boolean contains(int value)
     {
-        for (int index = 0; index < values.length; index++)
-        {
-            if (values[index] == value)
-            {
-                return true;
-            }
-        }
-
-        return false;
+        return valuesMap.contains(value);
```

Comment:
There are other places in the class where values are added or removed to the `values` array, so it looks like it will require more changes than just using the set in these current locations.

Reply:
Are you sure? All array modifications I could find in that class were related to the `newValues` array.